tests/bpf2c_tests/expected/map_dll.c

```diff
@@ -181,4151 +181,4151 @@ static uint16_t test_maps_maps[] = {
 #pragma code_seg(push, "xdp_prog")
 static uint64_t
 test_maps(void* context)
-#line 195 "sample/map.c"
+#line 195 "D:\\wrk\\ebpf-for-windows\\tests\\sample/map.c"
```

Comment:
Please use the script: .\scripts\generate_expected_bpf2c_output.ps1 to regenerate these files.

Reply:
@Alan-Jowett That command isn't documented anywhere under the docs directory.  Seems like we should document this somewhere.